tests/modules/vectorizer/integration_test_fastembed.py

```diff
@@ -0,0 +1,43 @@
+from dsp.modules.sentence_vectorizer import FastEmbedVectorizer
+import pytest
+
+from dspy.primitives.example import Example
+
+# Skip the test if the 'fastembed' package is not installed
+pytest.importorskip("fastembed", reason="'fastembed' is not installed. Use `pip install fastembed` to install it.")
+
+
+@pytest.mark.parametrize(
+    "n_dims,model_name", [(384, "BAAI/bge-small-en-v1.5"), (512, "jinaai/jina-embeddings-v2-small-en")]
+)
+def test_fastembed_with_examples(n_dims, model_name):
+    vectorizer = FastEmbedVectorizer(model_name)
```

Comment:
what do you think about mocking the `embed` function and keep it in the unit test?

the other question is does it require downloading of the models? if yeah I would say that would be good to be able to have it mocked and run it in the test in the CI without downloading.

Reply:
I planned tests. However, since the implementation is straightforward, the risk of failure comes from changes in the FastEmbed library, so these integration tests(Skipped if FastEmbed is not installed).

Reply:
So I think you don't want to include them in CI, am I right?

Reply:
I've added them to the CI in https://github.com/stanfordnlp/dspy/pull/831/commits/9b04c68f8eed9da0af6d7ac19b2a5e657f2f4d26.

Reply:
sorry for the delay in reply! 

I didn't meant to add them to the CI as they would take so long and I think they are not built for there to download each time we run the CI, just wanted to double check them for there with you! 

sorry if I made a confusion for you.

Reply:
Alright. I've removed it from the CI.